README.mediawiki

```diff
@@ -656,6 +656,8 @@ yum -y install krb5-workstation
 
 == Usage ==
 
+{{note}} The migration to ZenPackLib in version 2.6 of this ZenPack introduced an issue leading to apparent data loss in RRD graphs for filesystems, interfaces, and other components.  Since ZenPackLib uses a different directory heirarchy for RRD storage, component graphs after upgrade became empty (since the file paths changed).  Beginning in 2.7.0, an attribute called "windows_using_legacy_rrd_paths" is set to True during ZenPack installation if the previously installed version is earlier than 2.6.  This variable may need to be manually set (via zendmd) in rare scenarios such as rebuilding a Zenoss 4.x system from scratch (after updating from 2.5 to 2.7 of this ZenPack).  Also note that this issue does not affect Zenoss 5 and above.
```

Comment:
I don't think it's a good idea to even document dmd.windows_using_legacy_rrd_paths. I don't think there are even rare scenarios in which someone would change its value.